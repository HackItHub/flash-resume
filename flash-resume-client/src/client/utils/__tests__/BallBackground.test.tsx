import React from "react";
import renderer from "react-test-renderer";
import BallBackground from "../BallBackground";

it("renders ball background", () => {
  const component = renderer.create(<BallBackground />);
  const tree = component.tgoJSON();
  expect(tree).toMatchSnapshot();
});
