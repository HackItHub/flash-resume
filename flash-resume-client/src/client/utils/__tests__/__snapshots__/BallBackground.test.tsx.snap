// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ball background 1`] = `
<div
  className="ball-background-container h-full w-full absolute left-0 top-0"
>
  <div
    className="ball-background relative w-full h-full"
  />
</div>
`;
